### ISSUE TITLE
emb-m-05 - Application Window

### ISSUE THREAD

**Issue by bertramlyons** [LibraryOfCongress/embARC#22]
emb-m-05	Application Window	Yes	1-Critical	

Practically non-existent focus indicators. Keyboard users are unable to see which elements currently have focus, rendering keyboard access to much of the user interface impossible.	

Ensure clearly visible focus indicators with at least 3:1 contrast ratio with control and background.

**Comment by bertramlyons** [comment]
In progress. General focus indicators added.

**Comment by bertramlyons** [comment]
Completed and merged in version 1.3.0.
